docs/pages/api-docs/x-grid.md

```diff
@@ -59,6 +59,7 @@ import { XGrid } from '@material-ui/x-grid';
 | <span class="prop-name">onColumnHeaderEnter</span> | <span class="prop-type">(param: GridColumnHeaderParams, event: React.MouseEvent) => void</span> |   | Callback fired when a mouse enter event comes from a column header element. |
 | <span class="prop-name">onColumnHeaderLeave</span> | <span class="prop-type">(param: GridColumnHeaderParams, event: React.MouseEvent) => void</span> |   | Callback fired when a mouse leave event comes from a column header element. |
 | <span class="prop-name">onColumnOrderChange</span> | <span class="prop-type">(param: GridColumnOrderChangeParams, event: React.MouseEvent) => void</span> |   | Callback fired when a column is reordered. |
```

Comment:
Why do the 2 api pages differ on `onColumnOrderChange ` and `onColumnReorder` ? 
cc @m4theushw ?

Reply:
They are the same now